tests/test_backends.py

```diff
@@ -129,9 +129,14 @@ def test_decode_hmac_with_expiry(self):
 
     def test_decode_hmac_with_invalid_sig(self):
         self.payload['exp'] = aware_utcnow() + timedelta(days=1)
-        token_1 = jwt.encode(self.payload, SECRET, algorithm='HS256')
+
+        token_1 = jwt.encode(self.payload.copy(), SECRET, algorithm='HS256')
```

Comment:
Is there a reason we need to copy now? If so, then please use a property instead.